OxfordBodleian/Ullendorff/BDLeu5.xml

```diff
@@ -141,25 +277,24 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         
                     </physDesc>
                     
-                <!--    <history>
+                    <history>
                         <origin>
-                            <origDate when="" evidence=""/>
+                            <origDate notBefore="1700" notAfter="1750" evidence="lettering"/>
                         </origin>
-                        <provenance>  </provenance>
-                    </history>-->
-                    
-                    <additional>
-                        <adminInfo>
-                            <recordHist>
-                                <source>
-                                  <!--  <listBibl type="secondary">
-                                        <bibl></bibl>
-                                    </listBibl>-->
-                                </source>
-                            </recordHist>
-                        </adminInfo>
-                    </additional>
-                    
+                        <provenance>
+                            The provenance of the manuscript is unknown. It is unclear whether Gabra ʾƎgziʾabǝḥer (lit. 'the servant of God'), mentioned in the supplication 
```

Comment:
I would maybe minimally change the wording: From reading only this description, without consulting the encoding or manuscript, I would not have understood the exact situation in the supplication. Maybe something like:
`The supplication...mentions Gabra ʾƎgziʾabǝḥer (lit. 'the servant of God'), followed by an erasure, over which the name of a certain <persName ref="PRS14014Henok" role="owner">ʾAbuna Henok</persName> has been added. It is unclear....`

Reply:
Good, I have now revised the wording following your suggestion. Good idea to wait for other possible attestations of this name when the entire MS is accessible to us.
Thank you